### DISCUSSION TITLE
FieldNames or FieldName types are deprecated?

### DISCUSSION THREAD

**Discussion by jin60641** [react-hook-form/react-hook-form#2732] [Q&A] [answered]
I'm trying to upgrade react-hook-form's version to 6.5.3 from 4.4.8.
I used `import { FieldName } from 'react-hook-form'`, it does not work after the upgrade.
`dist/types/form.d.ts` is not exporting `FieldName` or `FieldNames` type.
How can I use that type definitions?
Thank you for reading.

**Comment by bluebill1049** [comment, marked as answer]
hey @jin60641 we will expose all the types in the following release: https://github.com/react-hook-form/react-hook-form/issues/2073

**Reply by jin60641** [reply]
Thanks :+1:  sorry for not checking issues

**Reply by bluebill1049** [reply]
not a problem at all.
